CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - In case the library contains empty entries, they are not written to disk. [#8645](https://github.com/JabRef/jabref/issues/8645)
 - The formatter `remove_unicode_ligatures` is now called `replace_unicode_ligatures`. [#9890](https://github.com/JabRef/jabref/pull/9890)
 - We improved the error message when no terminal was found [#9607](https://github.com/JabRef/jabref/issues/9607)
+- We changed and removed some illegal tags in javadoc. [koppor#475](https://github.com/koppor/jabref/issues/475)
```

Comment:
Changelog is intended for end users, so it's not necessary if it's only developer related stuff

Reply:
I got it. Thank you.

Reply:
The described way is the natural way to go for contirbutions on GitHub. a) there is no confustion, where `main` belongs to. Is it your updates or the master branch of the remote repository. b) you can do two independend changes in parallel.

I hope that your new PR https://github.com/JabRef/jabref/pull/9986 is the last one opened for that regard. Please push updates to your branch `fix-issue-475`. They will appear automatically!

Reply:
Hi @koppor , This is my first time contributing to open source project, sorry for the inconvience I brought to you. I want to make sure that I get you correctly: The natural way of contributing on Github is directly create a branch from repo (JabRef), rather than folking and manually creating PR request every time. As far as I know, changes pushed to branches in folked repo cannot trigger the PR check automatically.